Vagrantfile

```diff
@@ -44,7 +44,7 @@ fail 'Non-unique hostnames' if @network.collect { |_, v| v[:hostname] }.uniq!
 fail 'Non-unique ports' if @network.collect { |_, v| v[:ports].keys }.flat_map { |v| v }.uniq!
 
 def default_omnibus(config)
-  config.omnibus.chef_version = '12'
+  config.omnibus.chef_version = '14.5.27'
```

Comment:
why so specific?

Reply:
On line 355 we have defined the chef-client URL for windows forwarder. We will need to update that as well.
https://github.com/cerner/cerner_splunk/blob/00aa73d9b1f72969dd9963daeff3349a760cbfe5/Vagrantfile#L354-L355

Reply:
I had to be specific, the vagrant runs were failing when I only specified `14`.

Reply:
@nmattam fixed in 3ee2707

Reply:
What was the reason for the failure? That seems an issue that needs to be addressed if we cannot just pull the latest chef 14 client...

Reply:
We should figure out where that message is coming from since invoking the chef install script directly from a container seems to taking `12` and `14` and resolving the latest version of 12 and 14 just fine: 

```
[root@84cf013a9463 /]# curl https://www.chef.io/chef/install.sh -O; sh install.sh -v 14
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 23082  100 23082    0     0  77050      0 --:--:-- --:--:-- --:--:--  256k
el 6 x86_64
Getting information for chef stable 14 for el...
downloading https://omnitruck-direct.chef.io/stable/chef/metadata?v=14&p=el&pv=6&m=x86_64
  to file /tmp/install.sh.192/metadata.txt
trying curl...
sha1	dc1cfb5bee1e6f0df01c80b55ff7cd2fa22abc7e
sha256	219167dc31fcb1cef1fdd669403ccd91066ca23cf75e74da12e144c4b816fda6
url	https://packages.chef.io/files/stable/chef/14.5.33/el/6/chef-14.5.33-1.el6.x86_64.rpm
version	14.5.33
downloaded metadata file looks valid...
downloading https://packages.chef.io/files/stable/chef/14.5.33/el/6/chef-14.5.33-1.el6.x86_64.rpm
  to file /tmp/install.sh.192/chef-14.5.33-1.el6.x86_64.rpm
trying curl...
^C

[root@84cf013a9463 /]# curl https://www.chef.io/chef/install.sh -O; sh install.sh -v 12
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 23082  100 23082    0     0  84480      0 --:--:-- --:--:-- --:--:--  262k
el 6 x86_64
Getting information for chef stable 12 for el...
downloading https://omnitruck-direct.chef.io/stable/chef/metadata?v=12&p=el&pv=6&m=x86_64
  to file /tmp/install.sh.225/metadata.txt
trying curl...
sha1	ecb40884ed9fc7064eb04930c0695757e77aab8f
sha256	cf3c41d029c24ede942fa9c756990407b0396197ec6422df852028493bd6e3ef
url	https://packages.chef.io/files/stable/chef/12.22.5/el/6/chef-12.22.5-1.el6.x86_64.rpm
version	12.22.5
downloaded metadata file looks valid...
downloading https://packages.chef.io/files/stable/chef/12.22.5/el/6/chef-12.22.5-1.el6.x86_64.rpm
  to file /tmp/install.sh.225/chef-12.22.5-1.el6.x86_64.rpm
trying curl...
^C
```

Reply:
It likes a integer not a string i guess.

Reply:
fixed in: c9f3f7d